package.json

```diff
@@ -142,26 +139,30 @@
     "jest-resolve": "23.6.0",
     "jest-styled-components": "^6.3.1",
     "mini-css-extract-plugin": "0.4.3",
-    "optimize-css-assets-webpack-plugin": "5.0.8",
+    "optimize-css-assets-webpack-plugin": "^6.0.0",
     "pnp-webpack-plugin": "1.1.0",
     "postcss-flexbugs-fixes": "4.1.0",
     "postcss-loader": "3.0.0",
     "postcss-preset-env": "6.0.6",
     "postcss-safe-parser": "4.0.1",
-    "react-dev-utils": "^11.0.4",
+    "react-dev-utils": "^12.0.0",
     "react-render-html": "^0.6.0",
     "react-test-renderer": "^16.6.3",
     "resolve": "1.8.1",
-    "sass-loader": "^12.1.0",
+    "sass-loader": "^10.1.1",
```

Comment:
Why was downgrading needed here?

Reply:
Currently the `npm i` in the package is broken bec `sass-loader:12.x` needs webpack 5 https://github.com/webpack-contrib/sass-loader/blob/v12.1.0/package.json#L44
if you checkout the repo locally, it's not building.